### PR TITLE
Unify parameter notation for all scalers

### PR DESCRIPTION
**Pull request by JorTurFer**
This PR unifies parameter notation for all scalers. The pattern that all the parameters match is:
- name - Description. (Values: `x`, `y`, `z`, Default: `y`, Optional, Extra Info)

If a parameter is required or doesn't have defined/default values, the missing info has been removed from the pattern 

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda-docs/issues/540

### CODE REVIEW

**Comment by JorTurFer** [comment]
Please, take a look at this with calm, It was a bit repetitive and boring process and despite my revisions, I could do some mistakes

File: content/docs/1.4/scalers/azure-monitor.md [hunk 1]
```diff
@@ -29,17 +29,17 @@ triggers:
...
+- `metricFilter` - Is used to define a more specific part of the resource. You can filter by supported dimensions of the metric found in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported). (Optional)
+- `metricAggregationInterval` - Is the collection time of the metric. Reported in the format `"mm:hh:ss"`. (Default: `"0:5:0"`, Optional)
+- `metricAggregationType` - Is the aggregation method of the Azure monitor metric. Some possible values include `Average`, `Total`, `Maximum` with a full list in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported). No default.
+- `targetValue` - Is the target value for the Azure metric to use in the HPA.
+- `activeDirectoryClientId` - Is the name of the environment variable that contains the active directory client id. Should have the RBAC role of `Monitoring Reader`.
+- `activeDirectoryClientPassword` - Is the name of the environment variable that contains the active directory client password. Should have the RBAC role of `Monitoring Reader`.
```
Comment (tomkerkhove) [inline content/docs/1.4/scalers/azure-monitor.md:42]: Let's improve it a bit more
```suggestion
- `resourceURI` - Shortened URI to the Azure resource. The format is `"<resourceProviderNamespace>/<resourceType>/<resourceName>"`.
- `tenantId` - Tenant id for the Azure resource. Used for authentication.
- `subscriptionId` - Subscription id for the Azure resource. Used for determining the full resource URI.
- `resourceGroupName` - Resource group for the Azure resource.
- `metricName` - Name of the Azure Monitor metric. Must be an officially supported metric found in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported).
- `metricFilter` - Filter to define a more specific part of the resource. You can filter by supported dimensions of the metric found in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported). (Optional)
- `metricAggregationInterval` - Aggregation interval of the metric. Reported in the format `"mm:hh:ss"`. (Default: `"0:5:0"`, Optional)
- `metricAggregationType` - Aggregation method of the Azure monitor metric. Some possible values include `Average`, `Total`, `Maximum` with a full list in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported). No default.
- `targetValue` - Target value for the Azure metric to use in the HPA.
- `activeDirectoryClientId` - Name of the environment variable that contains the active directory client id. Should have the RBAC role of `Monitoring Reader`.
- `activeDirectoryClientPassword` - Name of the environment variable that contains the active directory client password. Should have the RBAC role of `Monitoring Reader`.
```

File: content/docs/1.4/scalers/huawei-cloudeye.md [hunk 2]
```diff
@@ -25,30 +25,30 @@ triggers:
...
-- `metricPeriod` is the granularity of the metric. This is not required. The default value is 300, unit is second.
+- `namespace` - Is the namespace of the metric.The format is service.item; service and item must be strings, must start with a letter, can only contain 0-9/a-z/A-Z/_, the total length of service.item is 3, the maximum is 32.
+- `metricName` - Is the name of the metric.
+- `dimensionName` - Is the dimension name of the metric.
+- `dimensionValue` - Is the dimension value of the metric.
+- `targetMetricValue` - Is the target value for your metric.
+- `minMetricValue` - Is the min value for your metric. If the actual value of the metric you get from cloudeye is less than the minimum value, then the scaler is not active.
+- `metricCollectionTime` - Is the collection time of the metric. Equivalent to the earliest start time of the end time. (Default: `300`, Optional)
+- `metricFilter` - Is the aggregation method of the metric. (Values: `max`, `min`, `average`, `sum`, `variance`, Default: `average`, Optional)
+- `metricPeriod` - Is the granularity of the metric. (Default: `300`, Optional)
```
Comment (tomkerkhove) [inline content/docs/1.4/scalers/huawei-cloudeye.md:36]: See feedback Azure Monitor

File: content/docs/1.5/scalers/artemis.md [hunk 3]
```diff
@@ -26,20 +26,20 @@ triggers:
...
-- `managementEndpoint`: "hostname:port" to connect to ActiveMQ Artemis management endpoint.
-- `queueName`: the name of the queue to check for the number of messages available.
-- `brokerName`: the name of the broker as defined in Artemis.
-- `brokerAddress`: the address name of the broker.
-- `queueLength` How much messages are in the queue. Default is 10. Optional.
+- `managementEndpoint` - "hostname:port" to connect to ActiveMQ Artemis management endpoint.
+- `queueName` - The name of the queue to check for the number of messages available.
+- `brokerName` - The name of the broker as defined in Artemis.
+- `brokerAddress` - The address name of the broker.
+- `queueLength` - How much messages are in the queue. (Default: `10`, Optional.)
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/artemis.md:33]:
```suggestion
- `managementEndpoint` - "hostname:port" to connect to ActiveMQ Artemis management endpoint.
- `queueName` - Name of the queue to check for the number of messages available.
- `brokerName` - Name of the broker as defined in Artemis.
- `brokerAddress` - Address name of the broker.
- `queueLength` - How much messages are in the queue. (Default: `10`, Optional.)
```

File: content/docs/1.5/scalers/azure-monitor.md [hunk 4]
```diff
@@ -29,17 +29,17 @@ triggers:
...
-- `metricAggregationInterval` is the collection time of the metric. Reported in the format `"mm:hh:ss"`. The default value is `"0:5:0"` (5 minutes). Optional.
-- `metricAggregationType` is the aggregation method of the Azure monitor metric. Some possible values include `Average`, `Total`, `Maximum` with a full list in the [official documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported). No default. Required.
-- `targetValue` is the target value for the Azure metric to use in the HPA. Required.
-- `activeDirectoryClientId` is the name of the environment variable that contains the active directory client id. Should have the RBAC role of `Monitoring Reader`. Optional. Required when `TriggerAuthentication` is not provided.
-- `activeDirectoryClientPassword` is the name of the environment variable that contains the active directory client password. Should have the RBAC role of `Monitoring Reader`. Optional. Required when `TriggerAuthentication` is not provided.
+- `resourceURI` - Is the shortened URI to the Azure resource. The format is `"<resourceProviderNamespace>/<resourceType>/<resourceName>"`.
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/azure-monitor.md:32]: See v1.4 comment

File: content/docs/1.5/scalers/cron.md [hunk 5]
```diff
@@ -24,10 +24,10 @@ triggers:
 
 **Parameter list:**
 
-- `timezone` is one of the acceptable values from the IANA Time Zone Database. The list of timezones can be found in: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones. Required.
-- `start` is the cron expression indicating the start of the cron schedule. Required.
-- `end` is the cron expression indicating the end of the cron schedule. Required.
-- `desiredReplicas` is the number of replicas to which the resource has to be scaled between the start and end of the cron schedule. Required.
+- `timezone` - Is one of the acceptable values from the IANA Time Zone Database. The list of timezones can be found in: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones.
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/cron.md:27]: See v1.4 comment for Azure Monitor

File: content/docs/1.5/scalers/huawei-cloudeye.md [hunk 6]
```diff
@@ -25,30 +25,30 @@ triggers:
...
-- `metricName` is the name of the metric. This is required.
-- `dimensionName` is the dimension name of the metric. This is required.
-- `dimensionValue` is the dimension value of the metric. This is required.
-- `targetMetricValue` is the target value for your metric. This is required.
-- `minMetricValue` is the min value for your metric. If the actual value of the metric you get from cloudeye is less than the minimum value, then the scaler is not active. This is required.
-- `metricCollectionTime` is the collection time of the metric. Equivalent to the earliest start time of the end time. This is not required. The default value is 300, unit is second.
-- `metricFilter` is the aggregation method of the metric. This is not required. The default value is average. Other values ​​include max, min, sum, variance.
-- `metricPeriod` is the granularity of the metric. This is not required. The default value is 300, unit is second.
+- `namespace` - Is the namespace of the metric.The format is service.item; service and item must be strings, must start with a letter, can only contain 0-9/a-z/A-Z/_, the total length of service.item is 3, the maximum is 32.
```
Comment (tomkerkhove) [inline content/docs/1.5/scalers/huawei-cloudeye.md:28]: See v1.4 comment for Azure Monitor

File: content/docs/2.1/operate/_index.md [hunk 7]
```diff
@@ -6,5 +6,5 @@ weight = 1
 
 We provide guidance & requirements around various areas to operate KEDA:
 
-- Cluster ([link](./cluster))
-- Integrate with Prometheus ([link](./prometheus))
\ No newline at end of file
+- Cluster ([link](./cluster)).
+- Integrate with Prometheus ([link](./prometheus)).
```
Comment (tomkerkhove) [inline content/docs/2.1/operate/_index.md:10]: No need to do this for listing imo
```suggestion
- Cluster ([link](./cluster))
- Integrate with Prometheus ([link](./prometheus))
```

File: content/docs/2.1/scalers/azure-service-bus.md [hunk 8]
```diff
@@ -31,11 +31,11 @@ triggers:
 **Parameter list:**
 
 - `messageCount` - Amount of active messages in your Azure Service Bus queue or topic to scale on.
-- `queueName` - Name of the Azure Service Bus queue to scale on. (optional)
-- `topicName` - Name of the Azure Service Bus topic to scale on. (optional)
-- `subscriptionName` - Name of the Azure Service Bus queue to scale on. (optional, required when `topicName` is specified)
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/azure-service-bus.md:36]: I would keep this part on required as this is important

File: content/docs/2.1/scalers/azure-service-bus.md [hunk 9]
```diff
@@ -31,11 +31,11 @@ triggers:
...
-- `namespace` - Name of the Azure Service Bus namespace that contains your queue or topic. (optional, required when pod identity is used)
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/azure-service-bus.md:37]: I would keep this part on required as this is important

File: content/docs/2.1/scalers/influxdb.md [hunk 10]
```diff
@@ -30,14 +30,14 @@ triggers:
...
-- `organizationName` is the organization name needed for the client to locate all information contained in that [organization](https://docs.influxdata.com/influxdb/v2.0/organizations/) such as buckets, tasks, etc.
-- `organizationNameFromEnv` defines the organization name, similar to `organizationName`, but reads it from an environment variable on the scale target.
-- `serverURL` holds the url value of the InfluxDB server.
-- `thresholdValue` is provided by the user. This value can vary from use case to use case depending on the data of interest, and is needed to trigger the scaling in/out depending on what value comes back from the query.
-- `query` is the flux query that will yield the value for the scaler to compare the `thresholdValue` against.
-- `metricName` an optional name to assign to the metric. If not set KEDA will generate a name based on masked version of the server url and organization name. If using more than one trigger it is required that all `metricName`(s) be unique.
+- `authToken` - Field holds the authentication token needed for the InfluxDB client to communicate with an associated server.
```
Comment (tomkerkhove) [inline content/docs/2.1/scalers/influxdb.md:33]: See v1.4 comment on Azure Monitor

**Review by tomkerkhove — COMMENTED** [review]
Nice work, first batch of comments (not fully reviewed yet but you get the idea)

**Comment by JorTurFer** [comment]
> Nice work, first batch of comments (not fully reviewed yet but you get the idea)

Yes I got it :) 
I have already changed the code following the suggestions. My only concern is that the "pattern" has changed a bit, now is something like:
- name - Description. (Values: `x`, `y`, `z`, Default: `y`, Optional, Extra Info)
I need the field "Extra info" to add some cases where we need to explain when the field is optional

Related with the reviewing process, no rush. There are a lot of smalls changes...

File: archetypes/scaler.md [hunk 11]
```diff
@@ -19,15 +19,15 @@ triggers:
 
 **Parameter list:**
 
-- `namespace` is the namespace of the metric. The format is service.item; service and item must be strings, must start with a letter, can only contain 0-9/a-z/A-Z/_, the total length of service.item is 3, the maximum is 32. (required)
+- `namespace` -Namespace of the metric. The format is service.item; service and item must be strings, must start with a letter, can only contain 0-9/a-z/A-Z/_, the total length of service.item is 3, the maximum is 32.
```
[the remark this answers is not in the document]
Comment (JorTurFer) [reply archetypes/scaler.md]: Fixed

**Comment by JorTurFer** [comment]
I have merged master into this branch and I updated the format from the newest commits. From my side, I think that this is ready to be merged :)

**Comment by tomkerkhove** [comment]
Thanks a ton for this big alignment!

**Comment by JorTurFer** [comment]
You are welcome!! ^^

**Pull request merged**